src/solvers/dgmulti/dg.jl

```diff
@@ -158,6 +158,29 @@ function max_dt(u, t, mesh::DGMultiMesh,
   return 2 * dt_min / (polydeg + 1)
 end
 
+function max_dt(u, t, mesh::DGMultiMesh,
+                constant_speed::Val{true}, equations, dg::DGMulti{NDIMS}, cache) where {NDIMS}
```

Comment:
This is to allow one to use the step size callback, right? For the error based time stepping this routine is not called? I ask because I see that it has a `constant_speed` set to true. If one uses variable wavespeed you would need to pass the `max` of the wave speed or something.

Reply:
Okay, that is what I figured. It is such a specific function we do not really need it without a use case. And even so, it could probably live in the elixir where it is needed (I think).